src/Command.js

```diff
@@ -166,6 +207,19 @@ export default class Command {
     }
   }
 
+  _legacyOptions() {
+    let opts = {};
+    if (this.name === "bootstrap" && (this.repository.asiniJson || {}).bootstrapConfig) {
```

Comment:
i think this could be part of `repository.asiniJson`, always ensuring an object is returned.